usagov/vote-gov-nvrf-app
# Vote-406: Fix Edit button wrapping on confirm page

## Jira ticket

[Vote-406](https://cm-jira.usa.gov/browse/VOTE-406)

## Description

This PR will fix a wrapping issue on the confirm page... issue was the edit button wrapping under the header, with this PR both the header and button will stay in line with each other. 

Solution:
I added a `class=grid-row` for both elements and took out the edit button from the H2 and wrapped that button in its own div and named it `class=edit-btn`.  I also added styles to that div and added a breakpoint to address when the user would be on mobile. 


## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

n/a

### QA/Test

1. visit [preview]() and fill out form to the confirm page 
2. adjust width and ensure that edit and header stay in line with each other 
3. review form and ensure there is no other regression seen 

## Checklist for the Developer




## Checklist for the Peer Reviewers
